.github/renovate.json

```
@@ -21,5 +21,15 @@
       ],
       "matchUpdateTypes": ["minor", "patch"]
     }
+  ],
+  "regexManagers": [
+    {
+      "fileMatch": ["\\proxy-actions/codecov/action.yml$"],
```

Just a copy/paste typo.  I tested with this update and it correctly finds the update 😃 

```suggestion
      "fileMatch": ["proxy-actions/codecov/action.yml$"],
```